File: 1icpc_contest/E.py
t = int(input())
T = 1
while T <= t:
    n = int(input())
    a = list(map(int, input().split()))[:n]
    count = 0
    s = 0
    for i in range(n):
        for j in range(i + 1, n - 1):
            if a[i] > a[j]:
                temp = a[j]
                a[j] = a[i]
                a[i] = temp
                count += 1
    for i in a:
        if i > 0:
            s += i
    print(f"Case {T}: {s} {count}")
    T += 1

File: 1icpc_contest/E.cpp
#include <iostream>
using namespace std;
int main(){
    int n,t,T=1,a[1000],i,j,temp;
    cin>>t;
    while(T<=t){
        cin>>n;
        for(i=0;i<n;i++)
            cin>>a[i];
        int count=0,sum=0;
        for(i=0;i<n;i++){
            for( j=i+1;j<n;j++){
                if(a[i]>a[j]){
                    temp=a[j];
                    a[j]=a[i];
                    a[i]=temp;
                    count++;
                }
            }
        }
        for(i=0;i<n;i++){
            if(a[i]>0) sum+=a[i];
        }
        cout<<"Case "<<T<<": "<<sum<<" "<<count<<endl;
        T++;
    }
    return 0;
}

